Admin/Admin.php

```diff
@@ -758,7 +753,7 @@ protected function buildShow()
         }
 
         $this->show = new FieldDescriptionCollection();
-        $mapper = new ShowMapper($this->showBuilder, $this->show, $this);
+        $mapper     = new ShowMapper($this->showBuilder, $this->show, $this);
```

Comment:
@rande well, this is maybe not as attempted.

This is due to `align_equals` that are align both constant and variable assignation.

Unfortunately, there is no `align_constant_equals`.

At least, I propose this solution:

* deactivate `unalign_double_arrow` and `unalign_equals` Symfony fixers. (already done)
* activate only `align_double_arrow` for array. Variables and constant will be kept "as is"

What do you think?